src/open_producten/producttypen/tests/test_onderwerp_admin.py

```diff
@@ -0,0 +1,94 @@
+from django.core.exceptions import ValidationError
+from django.forms import modelformset_factory
+from django.test import TestCase
+
+from open_producten.utils.tests.helpers import build_formset_data
+
+from ..admin.onderwerp import OnderwerpAdmin, OnderwerpAdminForm, OnderwerpAdminFormSet
+from ..models import Onderwerp
+from .factories import OnderwerpFactory
+
+
+def create_form(data, instance=None):
+    return OnderwerpAdminForm(
+        instance=instance,
+        data=data,
+    )
+
+
+class TestOnderwerpAdminForm(TestCase):
+
+    def setUp(self):
+        self.data = {
+            "naam": "test",
+            "_position": "first-child",
+            "path": "00010001",
+            "numchild": 1,
+            "depth": 1,
+        }
+
+    def test_hoofd_onderwerpen_must_be_published_when_publishing_sub_onderwerp(self):
```

Comment:
This publishing logic is set on the model level, so I think it should be tested there. Plus its easier to understand Model objects than form data